main.go

```diff
@@ -200,7 +219,13 @@ Generate an identity seed and launch a gateway:
 			Name:    "ipns-max-cache-ttl",
 			Value:   0,
 			EnvVars: []string{"RAINBOW_IPNS_MAX_CACHE_TTL"},
-			Usage:   "Optional cap on caching duration for IPNS/DNSLink lookups. Set to 0 to respect original TTLs.",
+			Usage:   "Optional cap on caching duration for IPNS/DNSLink lookups. Set to 0 to respect original TTLs",
+		},
+		&cli.BoolFlag{
+			Name:    "accelerated-dht",
```

Comment:
It seems like either we should specify here how this interacts with the `dht-routing` flag or just combine them into one string flag rather than 2 bools.

Reply:
@aschmahmann I'm up to change it to a string. Any suggestion of the possible values? `standard`, `accelerated` and... `off`?